custom_components/alfen_wallbox/manifest.json

```diff
@@ -9,5 +9,5 @@
     "issue_tracker": "https://github.com/leeyuentuen/alfen_wallbox/issues",
     "requirements": [],
     "config_flow": true,
-    "version": "2.9.2-beta-7"
+    "version": "2.9.2-beta-8"
```

Comment:
Next time, we could keep the 2.9.2 or 2.9.3 without beta or rc. Then we don't need to change them everytime.

The flow is beta, rc
Once we have a rc, we will put for the next release one digit higher. So normally this should be not beta8 anymore but 2.9.3

Reply:
Ok, I’ll change it to 2.9.3 ok?